app/models/kondo.rb

```diff
@@ -3,6 +3,24 @@ class Kondo < ApplicationRecord
   has_many :users, through: :bookings
   has_many :bookings # no dependent: :destroy, provider can't delete services with pending bookings
   has_one_attached :photo
+  acts_as_taggable_on :tags
 
+  include PgSearch::Model 
+    pg_search_scope :global_search,
+      against: [ :prefecture ],
+      associated_against: {
+        tags: [:name]
+      },
+      using: {
+        tsearch: {any_word: true}
+      }
+
+  $categories = ["designers", "keepers", "others"]
+  $prefectures = %w[Aichi Akita Aomori Chiba Ehime Fukui Fukuoka Fukushima Gifu Gunma Hiroshima
```

Comment:
I was thinking that maybe here we can shorten the list (just like we did in the seeds) and just use 5-6 prefectures max for the sake of simplicity :)

Reply:
Yeah I was also thinking about that, Using all of prefecture doesnt really make sense for us if we have only 10 providers haha

Reply:
Chose your favorite prefectures and we will adapt the seeds accordingly haha

Reply:
The reason behind the no result part is because javascript automatically submit the form if there's no data in the search form, but I already put the condition to show all for that, so whyyyy it isnt workinggg

Reply:
tbh it's working pretty well and I think we can push this branch already :) then tomorrow we can grab a teacher and ask for help!

Reply:
Fixed it I was I forgot to exclude empty strings in the array haha